payjoin-directory/src/config.rs

```diff
@@ -17,6 +17,27 @@ pub struct Config {
     pub timeout: Duration,
     pub storage_dir: PathBuf,
     pub ohttp_keys: PathBuf, // TODO OhttpConfig struct with rotation params, etc
+    #[cfg(feature = "acme")]
+    pub acme: Option<AcmeConfig>,
+}
+
+#[cfg(feature = "acme")]
+#[derive(Debug, Clone, Deserialize)]
+pub struct AcmeConfig {
+    pub domain: String,
+    pub contact: String,
+    pub lets_encrypt_production: bool,
+    pub cache_dir: PathBuf, // TODO Option?
```

Comment:
requiring someone to set it is simpler and ok to me. Makes sense to leave a reference in the `README`.